core/modules/telemetry/telemetry.info

```diff
@@ -0,0 +1,8 @@
+name = Telemetry
+description = Collects anonymous usage information to help improve Backdrop CMS and contributed modules.
+package = System
+version = BACKDROP_VERSION
+type = module
+backdrop = 1.x
+
+configure = admin/config/system/telemetry
```

Comment:
This links to `admin/config/system` in the PR sandbox.

Reply:
I recommend removing the `configure =` line since there is no configuration at this point.

Reply:
Yep, sorry this was from a previous iteration where you could select what telemetry data to send.